package.json

```diff
@@ -1,5 +1,15 @@
 {
-  "private": true,
+  "name": "parse-dashboard",
+  "description": "The parse dashboard",
+  "version": "1.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ParsePlatform/parse-dashboard"
+  },
+  "license": "BSD-3-Clause",
```

Comment:
We'll need to reference the `license` file. 

`"license: "LicenseRef-LICENSE"` 

Ref: https://github.com/npm/npm/issues/8795